src/components/dashboard/FaceVerification/api/FaceVerificationApi.js

```diff
@@ -29,7 +29,7 @@ class FaceVerificationApi {
   constructor(rootApi: typeof API, zoomApi: Axios, logger: any) {
     this.rootApi = rootApi
     this.zoomApi = zoomApi
-    this.logger = logger
+    this.logger = logger.child({ from: 'FaceVerificationAPI' })
```

Comment:
child instance is already injecting: https://github.com/GoodDollar/GoodDAPP/blob/1b183235f65f47eb36920c349fc9ac358c4c7004/src/components/dashboard/FaceVerification/api/FaceVerificationApi.js#L118

```suggestion
    this.logger = logger
```